server/test/mapping/mapping.test.js

```diff
@@ -0,0 +1,14 @@
+const app = require('../../app');
+const supertest = require('supertest');
+const request = supertest(app);
+
```

Comment:
@pcgamer1 can you follow same package structure as src for tests? It's really difficult to map which file is test for which. It's a common practice to have same src and test package structure.

either call these configRouter.test.js or rename your source file to mapping something